src/e2e-test/features/pipeline.triggers.feature

```diff
@@ -32,7 +32,47 @@ Feature: Pipeline Triggers
     Then Open inbound trigger and set a complex trigger when "trigger_test_pipeline_1" succeeds
     Then Visit pipeline "trigger_test_pipeline_1"
     Then Ensure outbound trigger to "trigger_test_pipeline_2" exists
+    Then Run down arrow is clicked
+    And Enter deployed runtime argument value "file:/tmp/cdap-ui-integration-fixtures/airports.csv" in row "0"
+    And Enter deployed runtime argument value "/tmp/cdap-ui-integration-fixtures" in row "1"
+    And Save deployed arguments button is clicked
+    And Wait for runtime arguments dialog to close
+    Then Run the pipeline
+    Then Deployed pipeline status is "Succeeded"
     Then Visit pipeline "trigger_test_pipeline_2"
+    Then Deployed pipeline status is "Succeeded"
```

Comment:
this needs to check twice?

Reply:
i think the first one is checking pipeline1 run success

Reply:
correct, this is checking pipeline2 was successfully triggered by pipeline1 and run succeeds